tests/javaobject/PdxTests/PdxType.java

```diff
@@ -102,21 +103,21 @@ public void init()
   {
     m_char = 'C';
     m_bool = true;
-    m_byte = (byte)0x74;
+    m_byte = (byte)0xa4;
```

Comment:
What's the story behind changing these magic numbers?  Was there any significance to their choosing in the first place?  They don't appear to correspond to any kind of MAX_* values or things that test edge cases or anything, so why change 'em?

Reply:
are we just buying ourselves an extra 48 hours...

Reply:
This indeed warrants some explanation. The PdxType.java is the Java equivalent of PdxType.cs in the clicache tests. (Actually, since java came first it may be better to say PdxType.cs is the C# equivalent of PdxType.java. Note that PdxType.java has a pair of variables for many types (m_byte and m_sbyte, m_int16Array and m_uint16Array, for example). But each member of the pair is a signed type, as it must be since Java has no unsigned types.

Since this PR adds support for unsigned types in the .NET client, it seemed prudent to use unsigned values for the constants which would appear as negative numbers in Java to differentiate them from signed values. This provides for better testing of our unsigned APIs in my opinion.